src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -31,6 +31,10 @@ internal class AzureAppConfigurationProvider : ConfigurationProvider, IConfigura
         private AzureAppConfigurationOptions _options;
         private ConcurrentDictionary<string, ConfigurationSetting> _settings;
 
+        private static TimeSpan? MinCacheExpirationTime;

```

Comment:
>TimeSpan [](start = 23, length = 8)

Do we need to cache this thing at the class level? If so then it could probably be set in the constructor.